src/functions/schema.ts

```diff
@@ -13,6 +12,8 @@ const betterAjvErrors = require('better-ajv-errors/lib/modern');
 
 export interface ISchemaOptions {
   schema: object;
+  // The oasVersion, either 2 or 3 for OpenAPI Spec versions, or undefined / 0 for JSON Schema
```

Comment:
```suggestion
  // The oasVersion, either 2 or 3 for OpenAPI Spec versions, or null / 0 for JSON Schema
```
Let's go with null instead of 0.
Thoughts?

Reply:
I chose 0 so that it can be used in the ajvInstances object as key (and because I was confused with how to enforce null as default value in TS).

Reply:
@P0lip I couldn't get your suggestions working and I'm not familiar with how Optional works, so I tried a bit and the solution committed now works without 0. Please check whether that's okay for you.